x-pack/plugins/security/public/authentication/unauthorized/unauthorized_page.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton } from '@elastic/eui';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import type { CoreStart, IBasePath } from 'src/core/public';
+
+import { parseNext } from '../../../common/parse_next';
+import { AuthenticationStatePage } from '../components';
+
+interface Props {
+  basePath: IBasePath;
+}
+
+export function UnauthorizedPage({ basePath }: Props) {
```

Comment:
__note:__ once we agree on the approach I'll reach to the Design and Docs teams to have a more user-friendly UI and text.